docs/openapi-docs/src/test/scala/sttp/tapir/docs/openapi/VerifyYamlTest.scala

```diff
@@ -547,4 +547,19 @@ class VerifyYamlTest extends AnyFunSuite with Matchers {
     val actualYaml = OpenAPIDocsInterpreter().toOpenAPI(ep, "title", "1.0").toYaml
     noIndentation(actualYaml) shouldBe expectedYaml
   }
+
+  test("should contain description field for Option[Json] field") {
+    val expectedYaml = load("expected_type_and_description_for_circe_json.yml")
+
+    val myEndpoint = endpoint
+      .post
+      .in(jsonBody[ExampleMessageIn])
```

Comment:
it would be better to keep the example as self-contained as possible - maybe we can move the class over here and nest it inside the test?